.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
We're scanning go modules by Trivy, so this is unneeded

Reply:
Ok, I remove this part then